src/stores/settings.js

```diff
@@ -55,7 +55,7 @@ export const useSettingsStore = defineStore('settings', {
 
 	getters: {
 		getShowMediaSettings: (state) => (token) => {
-			if (state.showMediaSettings[token] !== undefined) {
+			if (!token || state.showMediaSettings[token] !== undefined) {
 				return state.showMediaSettings[token]
 			}
```

Comment:
If the purpose of this change is to return `undefined` when token is not set (because `state.showMediaSettings[undefined]` is probably `undefined`), I'd explicitly return false:
```js
if (!token) {
  return false
}
```

But does this method called somewhere without a token?

Reply:
It happened at root view, when MediaSettings is mounted (always), but there are no conversations joined.
But maybe would be better to make this check at Vue component

Reply:
Check for token here is also fine to me, but not to return `state.showMediaSettings[undefined]` to fallback to `false` value.